scripts/release/rpm/azure-cli.spec

```diff
@@ -54,12 +54,18 @@ for d in %{buildroot}%{cli_lib_dir}/bin/*; do perl -p -i -e "s#%{buildroot}##g"
 # Create executable
 mkdir -p %{buildroot}%{_bindir}
 python_version=$(ls %{buildroot}%{cli_lib_dir}/lib/ | head -n 1)
+
 printf "#!/usr/bin/env bash
 bin_dir=\`cd \"\$(dirname \"\$BASH_SOURCE[0]\")\"; pwd\`
 python_cmd=%{python_cmd}
 if command -v ${python_version} &>/dev/null; then python_cmd=${python_version}; fi
+%if 0%{?cblmariner}
+AZ_INSTALLER=RPM PYTHONPATH=\"%{_libdir}/az/lib/${python_version}/site-packages\" \$python_cmd -sm azure.cli \"\$@\"
```

Comment:
Why not use `%{_libdir}` for both?

Reply:
> The macro `%{_lib}` & `%{_libdir}` both point to `/usr/lib`.

This is against fedora's definition:

https://docs.fedoraproject.org/en-US/packaging-guidelines/RPMMacros/

![image](https://user-images.githubusercontent.com/4003950/142176721-c484b5a7-9abd-4760-bbc3-2ebfca88cfba.png)

Found this link from https://www.google.com/search?q=_libdir

Reply:
> This is against fedora's definition:


Sure, but since we have `Mariner` as a new distribution, we might not want to compare `fedora` & `mariner`.
Also, this is the reason, that we have to have an `%if - %else` condition.
You are absolutely correct, if we are speaking in terms of `fedora`. Given `mariner` is not `fedora` but an altogether different distribution, this should be acceptable IMO.
Please let me know your thoughts.

Reply:
Do you have a URL I can pull a Mariner image and have a test?

Reply:
All VHD/ISO/Container-Images can be found [here](https://eng.ms/docs/products/mariner-linux/gettingstarted/downloads).

Please feel free to reach out if anything :)

Reply:
I think `%{_lib}` is used here for a reason. Without it being available, we can't figure out the `PYTHONPATH` if the RPM is installed to some other place. See https://github.com/Azure/azure-cli/pull/17491#discussion_r759928226

Reply:
Let's move the entry script work to https://github.com/Azure/azure-cli/pull/20574.

Reply:
Sure

Reply:
Do you want me to rebase this PR with the one you created ? or should I modify this pr with the change `s/%{_lib}/lib64/` ?

Reply:
Please just don't change this file.

Reply:
@jiasli As discussed offline, copying the `az-cli.spec` changes from PR #20574.